File: Hayaa.Security.Service/Client/Java/hayaa.security.client/hayaa.worker.security.client/src/main/java/hayaa/workersecurity/client/LoginToken.java
package hayaa.workersecurity.client;

import java.util.Date;

class LoginToken {
    public Integer getJobTokenId() {
        return jobTokenId;
    }

    public void setJobTokenId(Integer jobTokenId) {
        this.jobTokenId = jobTokenId;
    }

    public Integer getJobId() {
        return jobId;
    }

    public void setJobId(Integer jobId) {
        this.jobId = jobId;
    }

    public Integer getUserId() {
        return userId;
    }

    public void setUserId(Integer userId) {
        this.userId = userId;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public Date getEndTime() {
        return endTime;
    }

    public void setEndTime(Date endTime) {
        this.endTime = endTime;
    }

    public Boolean getStatus() {
        return status;
    }

    public void setStatus(Boolean status) {
        this.status = status;
    }

    private Integer jobTokenId;
    private Integer jobId;
    private Integer userId;
    private String token;
    private Date endTime;
    private Boolean status;
}

File: v1/Hayaa.Security.Service/Model/LoginToken.cs
using Hayaa.BaseModel;
using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.Security.Service
{
   public class LoginToken : BaseData
    {
        public int LoginTokenId { set; get; }
        public int UserId { set; get; }
        public String Token { set; get; }
        public DateTime EndTime { set; get; }
        public Boolean Status { set; get; }
    }
}
